dask/dataframe/core.py

```diff
@@ -1662,6 +1666,19 @@ def meth(self, other, level=None, fill_value=None, axis=0):
         meth.__doc__ = op.__doc__
         bind_method(cls, name, meth)
 
+    @classmethod
+    def _bind_comparison_method(cls, name, comparison):
+        """ bind comparison method like DataFrame.add to this class """
+
+        def meth(self, other, level=None, fill_value=None, axis=0):
```

Comment:
Given the error message that you're seeing perhaps this `fill_value` keyword isn't appropriate?